lib/db.js

```diff
@@ -257,6 +257,16 @@ module.exports = (
 
     return P.all(promises)
       .spread((mysqlSessionTokens, redisTokens) => {
+        if (config.tokenLifetimes.sessionTokenWithoutDevice) {
```

Comment:
nit: do you want to store `config.tokenLifetimes.sessionTokenWithoutDevice` into a variable. Using it twice below?